packages/api/src/models/security/authentication/gdb-token-auth-provider.ts

```diff
@@ -0,0 +1,47 @@
+import {AuthenticatedUser} from '../authenticated-user';
+import {AuthStrategy} from './auth-strategy';
+import {AuthStrategyType} from './auth-strategy-type';
+import {MapperProvider, service} from '../../../providers';
+import {AuthenticatedUserMapper, AuthenticationStorageService, SecurityContextService} from '../../../services/security';
+import {SecurityRestService} from '../../../services/security/security-rest.service';
```

Comment:
This import doesn't look OK. Probably security-rest.service.ts is not exported through the index.ts file in the module. Check it out and fix the import pls